comid/referencevalue.go

```diff
@@ -1,24 +1,86 @@
-// Copyright 2021 Contributors to the Veraison project.
+// Copyright 2021-2024 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
 
 package comid
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
 
-type ReferenceValue struct {
+	"github.com/veraison/corim/extensions"
+)
+
```

Comment:
Now that we have a common triples semantics, best to add a comment to state that 

// ValueTriple is a common structure that carries the information for Reference Values and Endorsed Values as of V4 of the specification

Reply:
Thank you!